docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -15,15 +15,46 @@ Directories to cache
 
 In your CI's config file, we recommend caching these directories:
 
-- `$HOME/.cache/pants/setup`: the initial bootstrapping of Pants.
-- `$HOME/.cache/pants/named_caches`: caches of tools like pip and PEX.
-- `$HOME/.cache/pants/lmdb_store`: cached content for prior Pants runs, e.g. prior test results.
+
+- `$HOME/.cache/pants/setup`<br>
+  This is the Pants bootstrap directory. Cache this against the version, as specified
+  in `pants.toml`.  E.g., if using GitHub Actions you can use:
```

Comment:
The named caches are harder, but for this it seems we could provide a step or two that used python + toml to parse out the version and set it as an output for the cache step to use as the right most key instead. In fact, we could can that as an action in pants build/actions and just point folks to use that in this setup rec. For later.

Reply:
You beat me to it: https://github.com/pantsbuild/example-python/pull/104#discussion_r944977109

Reply:
Maybe, but let's get this in for now, as it's a big improvement on what we had before, and we're not trying to be everyone's devops people, just give a sense of what can profitably be cached.